source/Gemfile

```diff
@@ -33,6 +33,7 @@ gem 'spring',        group: :development
 gem 'bcrypt', '~> 3.1.7'
 
 # Use unicorn as the app server
+gem 'rspec-rails'
 # gem 'unicorn'
```

Comment:
Of all the places in this file to stick it, you split the comment about unicorn from the usage of unicorn‽ COME ON! GEEZ!